helm/opendistro-es/user-roles-values.yaml

```diff
@@ -0,0 +1,227 @@
+    opendistro_security.allow_unsafe_democertificates: false
+    opendistro_security.allow_default_init_securityindex: false
+    opendistro_security.audit.type: internal_elasticsearch
+    opendistro_security.enable_snapshot_restore_privilege: true
+    opendistro_security.check_snapshot_restore_write_privileges: true
+    cluster.routing.allocation.disk.threshold_enabled: false
+    opendistro_security.audit.config.disabled_rest_categories: NONE
+    opendistro_security.audit.config.disabled_transport_categories: NONE
+    opendistro_security.authcz.admin_dn:
+    - 'CN=admin.elastic-system.svc.cluster.local,O=Elastisys'
+    opendistro_security.nodes_dn:
+    - 'CN=nodes.elastic-system.svc.cluster.local,O=Elastisys'
+    opendistro_security.ssl.transport.pemcert_filepath: elk-transport-crt.pem
+    opendistro_security.ssl.transport.pemkey_filepath: elk-transport-key.pem
+    opendistro_security.ssl.transport.pemtrustedcas_filepath: elk-transport-root-ca.pem
+    opendistro_security.ssl.transport.enforce_hostname_verification: false
+
+  securityConfig:
+    enabled: true
+    config:
+      config.yml: |-
+        _meta:
+          type: "config"
+          config_version: 2
+        config:
+          dynamic:
+            kibana:
+              multitenancy_enabled: true
+              server_username: kibana
+
+            authc:
+              basic_internal_auth_domain:
+                description: "Authenticate via HTTP Basic against internal users database"
+                http_enabled: true
+                transport_enabled: true
+                order: 0
+                http_authenticator:
+                  type: basic
+                  challenge: false
+                authentication_backend:
+                  type: internal
+
+      #Define users, roles, mappings and tenants below
+      internal_users.yml: |-
+        _meta:
+          type: "internalusers"
+          config_version: 2
+
+        #define admin user
+        admin:
+          hash: "$2a$12$VcCDgh2NDk07JGN0rjGbM.Ad41qVR/YFJcgHp0UGns5JDymv..TOG"
+          reserved: true
+          backend_roles:
+          - "admin"
+          description: "Demo admin user"
+
+        #define user for  for the Kibana server
+        kibana:
+          hash: "$2y$12$efAwiXyV.uRLnYy9c0Kdju4hrr8fso2cKC7ObkHTrTk4jGXHlzmzi"
+          reserved: true
+          description: "kibana server user"
+
+        #define user for fluentd
+        fluentd:
+          hash: "$2a$12$u1ShR4l4uBS3Uv59Pa2y5.1uQuZBrZtmNfqB3iM/.jL0XoV9sghS2"
+          reserved: true
+          description: "fluentd user"
+
+        #define user for snapshots
+        snapshot:
+          hash: "$2y$12$HARtGIKFsoVT8ypHHA6QwOx6XPoP.En/ReIjLGI90rVqLHf2Lcmhe"
+          reserved: true
+          description: "manage_snapshot user"
+
+        #define user for ilm
+        ilm:
+          hash: "$2y$12$SHMJ3.DOHJFCVi6wOrLfh.gXcYAgh4dBtRitA0d/Txd2Z3tbSiGoy"
+          reserved: true
+          description: "ilm user"
```

Comment:
Is it necessary to set the passwords through hash? Currently the password is set from a Secret directly which is really nice. Would it be possible to do the same here?
In any case, we should not have the hashes directly here. Just add some placeholders or leave them empty, preferably so that chart installation fails unless they are set in some other way. Otherwise there is a risk that we use these passwords by mistake in all clusters.